PointOfSaleSystem.API/Repositories/EmployeeRepository.cs

```diff
@@ -0,0 +1,94 @@
+﻿using AutoMapper;
+using PointOfSaleSystem.API.Context;
+using PointOfSaleSystem.API.Models;
+using PointOfSaleSystem.API.Models.Entities;
+using PointOfSaleSystem.API.Models.Enums;
+using PointOfSaleSystem.API.Repositories.Interfaces;
+using PointOfSaleSystem.API.RequestBodies.Company;
+using PointOfSaleSystem.API.RequestBodies.Employee;
+using static Microsoft.EntityFrameworkCore.DbLoggerCategory.Database;
+
+namespace PointOfSaleSystem.API.Repositories
+{
+    public class EmployeeRepository : IEmployeeRepository
+    {
+        private readonly PointOfSaleSystemContext _context;
+        private readonly IMapper _mapper;
+
+        public EmployeeRepository(
+            PointOfSaleSystemContext context,
+            IMapper mapper)
+        { 
+            _context = context;
+            _mapper = mapper;
+        }
+
+        public void Create(Employee employee)
+        {
+            EmployeeEntity employeeEntity = _mapper.Map<EmployeeEntity>(employee);
```

Comment:
Are you sure that this mapper will work? you should add 
cfg.CreateMap<EmployeeEntity, Employee>();
cfg.CreateMap<Employee, EmployeeEntity>();

inside the Program.cs MapperConfiguration